modules/ting_search_ortograf/css/ting_search_ortograf_admin_form.css

```diff
@@ -0,0 +1,23 @@
+#ting-search-ortograf-admin .form-type-textfield {
```

Comment:
I imagine this stylesheet might be leftovers from `ting_search_autocomplete`. Is it really needed now? We do not have as many fields to manage.